src/scripts/components/LoginPage.tsx

```diff
@@ -10,7 +11,7 @@ import Spinner from './Spinner';
 export interface LoginProps {
   isLoggedIn: boolean;
   isPopup: boolean;
-  source: 'web-login' | 'install';
+  source: 'web-login' | 'install' | 'quickstart';
```

Comment:
Not a biggie but for consistency
```suggestion
  source: 'web-login' | 'install' | 'quick-start';
```